package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "czifui",
-  "version": "1.0.1",
+  "version": "0.0.0-development",
```

Comment:
turned out that Semantic Release ignores this and relies on commit history to determine the next version 😆 Reverting this back to `0.0.0-development` to avoid confusion